.github/dependabot.yml

```diff
@@ -0,0 +1,25 @@
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "daily"
+      time: "11:00"
+    assignees:
+      - "jawnsy"
+    reviewers:
+      - "jawnsy"
+    ignore:
+      # GitHub always delivers the latest versions for each major
+      # release tag, so handle updates manually
+      - dependency-name: "actions/*"
```

Comment:
@cdr/code-server-reviewers FYI, I made two minor changes to this since it was last reviewed:

1. Ignore all of the official GitHub Actions actions, we'll need to manage these manually
2. Format dependabot.yml with Prettier as CI needs that

Reply:
Sweet! Thanks for the update!

Reply:
@jsjoeio I'll need your review again, it seems your personal reviews don't count :raised_eyebrow: 

![image](https://user-images.githubusercontent.com/52710/110989779-39b41b00-8327-11eb-9ef7-76cc00a27664.png)

Reply:
Ah weird! Maybe it's cause you dismissed the stale one? I'll reapprove!
<img width="478" alt="image" src="https://user-images.githubusercontent.com/3806031/110990526-ac75c400-8330-11eb-8db3-5827a588ce31.png">

Reply:
interesting, I wonder if it did that automatically because I added commits, that'd make sense I guess!

Reply:
Thanks for the approval, excited to see how this goes... :crossed_fingers: